renovate.json

```diff
@@ -2,8 +2,10 @@
   "extends": ["group:recommended", "group:monorepos"],
   "reviewers": ["team:dev"],
   "rangeStrategy": "pin",
-  "node": {
-    "supportPolicy": ["active"],
-    "rangeStrategy": "bump"
-  }
+  "packageRules": [
+    {
+      "packageNames": ["node"],
+      "enabled": false
+    }
```

Comment:
So this is saying:

1. Find any package named "node"
2. Apply the following special rules to it:
3. The only special rule is to entirely disable renovate for it.

Is that right?

Reply:
Yes that is my understanding